### DISCUSSION TITLE
Should we add instructions to fix homebrew python/pip?

### DISCUSSION THREAD

**Discussion by CarlaAstudillo** [texastribune/data-visuals-guides#25] [General]
These [instructions](https://github.com/texastribune/data-visuals-guides/blob/3fc258bca8fcd5226075b78dce22db21d8b849de/computer-setup.md?plain=1#L145C1-L148C4) didn't really work the first time I ran it.

```sh
brew install python
pip install --upgrade setuptools pip wheel
```

We use homebrew to install python and then we use pip to install other packages, but when I ran it, it said pip wasn't there. I ended up using pip3 to install but not sure if it was the right call? Did a little more research and it said that it IS there, it's just that it needs to be [relinked](https://macpaw.com/how-to/install-pip-mac).

`brew unlink python && brew link python`

Did anyone else run into this problem and what was the solution? Should we add instructions on how to fix pip for a homebrew install?

Note: I usually use `pipenv` and `pyenv` to manage my environments and python versions respectively and rarely use my system-wide python anyway so that might be why I hadn't noticed a problem.

**Comment by csessig86** [comment]
Interesting, I don't have those same issues when I run `pip`. That said, I don't think we need to install `setuptools` or `wheel` because we don't use them, as far as I can tell. I'm leaning towards just removing that line altogether...
